src/pages/GiveParty/PartyProfile.tsx

```diff
@@ -1,21 +1,12 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
@wingfeatherwave any unsued vars still?

Reply:
there was one in the loading.  removed it